docs/src/modules/ROOT/pages/using-timefold-solver/benchmarking-and-tweaking.adoc

```diff
@@ -596,6 +596,27 @@ Useful for comparing different score calculators and/or constraint implementatio
 Also useful to measure the scalability cost of an extra constraint.
 
 
+[#benchmarkReportMoveEvaluationSpeedSummary]
+=== Move evaluation speed summary (graph And table)
+
+Shows the move evaluation speed: a count per move, per second and per problem scale for each solver configuration.
+
+Useful for comparing different solver algorithms,
+score calculators and/or constraint implementations
+(presuming that the solver configurations do not differ otherwise, including the move selector configuration).
+Also useful to measure the scalability cost of an extra constraint.
+
+[IMPORTANT]
+====
+When improving your move evaluation,
+it's important to note that comparing a configuration
+that uses xref:optimization-algorithms/move-selector-reference.adoc#ruinRecreateMoveSelector[Ruin and Recreate] moves with one that doesn't wouldn't be fair.
+This is because the configuration using Ruin and Recreate will likely execute fewer moves than one that doesn't.
+On the other hand, the Ruin and Recreate moves will likely calculate the score more times, improving the __score calculation speed__.
+The recreate step runs a construction heuristic,
+which uses greedy logic to find a better location to assign each one of the entities removed with the ruin step.
+====
```

Comment:
This note arguably applies to score calculation count, but is irrelevant in move count, no? Move counts are the same regardless of what moves are used.

(It is true that RnR is slow and therefore will decrease move count, but you can say that about any slow move. This is not specific to RnR.)

Reply:
Makes sense.